crash when trying to view PR

```
# logcat -b crash
01-09 09:21:00.005 28092 28092 E AndroidRuntime: FATAL EXCEPTION: main
01-09 09:21:00.005 28092 28092 E AndroidRuntime: Process: com.gh4a, PID: 28092
01-09 09:21:00.005 28092 28092 E AndroidRuntime: com.squareup.moshi.JsonDataException: Expected one of [completed, in_progress, queued, requested] but was waiting at path $.check_runs[2].status
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.squareup.moshi.StandardJsonAdapters$EnumJsonAdapter.fromJson(StandardJsonAdapters.java:258)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.squareup.moshi.StandardJsonAdapters$EnumJsonAdapter.fromJson(StandardJsonAdapters.java:227)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:137)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:137)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.meisolsson.githubsdk.model.$AutoValue_CheckRun$MoshiJsonAdapter.fromJson($AutoValue_CheckRun.java:96)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.meisolsson.githubsdk.model.$AutoValue_CheckRun$MoshiJsonAdapter.fromJson($AutoValue_CheckRun.java:33)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:137)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.squareup.moshi.CollectionJsonAdapter.fromJson(CollectionJsonAdapter.java:76)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.squareup.moshi.CollectionJsonAdapter$2.fromJson(CollectionJsonAdapter.java:53)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:137)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:137)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.meisolsson.githubsdk.model.$AutoValue_CheckRunsResponse$MoshiJsonAdapter.fromJson($AutoValue_CheckRunsResponse.java:42)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.meisolsson.githubsdk.model.$AutoValue_CheckRunsResponse$MoshiJsonAdapter.fromJson($AutoValue_CheckRunsResponse.java:21)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:137)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at retrofit2.converter.moshi.MoshiResponseBodyConverter.convert(MoshiResponseBodyConverter.java:45)01-09 09:21:00.005 28092 28092 E AndroidRuntime:at retrofit2.converter.moshi.MoshiResponseBodyConverter.convert(MoshiResponseBodyConverter.java:27)01-09 09:21:00.005 28092 28092 E AndroidRuntime:at retrofit2.OkHttpCall.parseResponse(OkHttpCall.java:223)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at retrofit2.OkHttpCall.execute(OkHttpCall.java:186)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at retrofit2.adapter.rxjava2.CallExecuteObservable.subscribeActual(CallExecuteObservable.java:45)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.Observable.subscribe(Observable.java:12267)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.internal.operators.observable.ObservableSingleSingle.subscribeActual(ObservableSingleSingle.java:35)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.Single.subscribe(Single.java:3603)01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.tspoon.traceur.SingleOnAssembly.subscribeActual(SingleOnAssembly.java:43)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.Single.subscribe(Single.java:3603)01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.internal.operators.single.SingleMap.subscribeActual(SingleMap.java:34)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.Single.subscribe(Single.java:3603)01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.tspoon.traceur.SingleOnAssembly.subscribeActual(SingleOnAssembly.java:43)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.Single.subscribe(Single.java:3603)01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.internal.operators.single.SingleMap.subscribeActual(SingleMap.java:34)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.Single.subscribe(Single.java:3603)01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.tspoon.traceur.SingleOnAssembly.subscribeActual(SingleOnAssembly.java:43)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.Single.subscribe(Single.java:3603)01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.internal.operators.single.SingleMap.subscribeActual(SingleMap.java:34)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.Single.subscribe(Single.java:3603)01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.tspoon.traceur.SingleOnAssembly.subscribeActual(SingleOnAssembly.java:43)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.Single.subscribe(Single.java:3603)01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.internal.operators.single.SingleZipArray.subscribeActual(SingleZipArray.java:63)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.Single.subscribe(Single.java:3603)01-09 09:21:00.005 28092 28092 E AndroidRuntime:at com.tspoon.traceur.SingleOnAssembly.subscribeActual(SingleOnAssembly.java:43)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.Single.subscribe(Single.java:3603)01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.Scheduler$DisposeTask.run(Scheduler.java:578)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:66)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:57)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at java.util.concurrent.FutureTask.run(FutureTask.java:266)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
01-09 09:21:00.005 28092 28092 E AndroidRuntime:at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at java.lang.Thread.run(Thread.java:764)
01-09 09:21:00.006 28092 28092 E AndroidRuntime: Caused by: com.tspoon.traceur.TraceurException: Debug Exception generated at call site
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at dalvik.system.VMStack.getThreadStackTrace(Native Method)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at java.lang.Thread.getStackTrace(Thread.java:1538)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at io.reactivex.Observable.singleOrError(Observable.java:11650)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at retrofit2.adapter.rxjava2.RxJava2CallAdapter.adapt(RxJava2CallAdapter.java:79)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at retrofit2.HttpServiceMethod.invoke(HttpServiceMethod.java:89)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at retrofit2.Retrofit$1.invoke(Retrofit.java:147)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at java.lang.reflect.Proxy.invoke(Proxy.java:1006)01-09 09:21:00.006 28092 28092 E AndroidRuntime:at $Proxy22.getCheckRunsForRef(Unknown Source)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.fragment.PullRequestConversationFragment.loadCommitStatusesIfOpen(PullRequestConversationFragment.java:489)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.fragment.PullRequestConversationFragment.onActivityCreated(PullRequestConversationFragment.java:127)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.fragment.app.Fragment.performActivityCreated(Fragment.java:2996)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.fragment.app.FragmentStateManager.activityCreated(FragmentStateManager.java:580)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:285)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2189)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:2100)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.fragment.app.FragmentManager.execSingleAction(FragmentManager.java:1971)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.fragment.app.BackStackRecord.commitNowAllowingStateLoss(BackStackRecord.java:311)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.fragment.app.FragmentStatePagerAdapter.finishUpdate(FragmentStatePagerAdapter.java:274)01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.viewpager.widget.ViewPager.populate(ViewPager.java:1244)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.viewpager.widget.ViewPager.populate(ViewPager.java:1092)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.viewpager.widget.ViewPager.setOffscreenPageLimit(ViewPager.java:856)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.BasePagerActivity.updateTabVisibility(BasePagerActivity.java:143)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.BasePagerActivity.invalidatePages(BasePagerActivity.java:61)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.BasePagerActivity.invalidateTabs(BasePagerActivity.java:65)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.activities.PullRequestActivity.lambda$load$6(PullRequestActivity.java:514)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.activities.PullRequestActivity.lambda$load$6$PullRequestActivity(Unknown Source:0)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.activities.-$$Lambda$PullRequestActivity$nFWg86hv32nRF6uhTT2bLAk52TI.accept(Unknown Source:4)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at io.reactivex.internal.observers.ConsumerSingleObserver.onSuccess(ConsumerSingleObserver.java:62)01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.philosophicalhacker.lib.ReactiveLoaders$LoaderSingle$1.onLoadFinished(ReactiveLoaders.java:55)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.philosophicalhacker.lib.RxLoaderCallbacks.onLoadFinished(RxLoaderCallbacks.java:35)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.loader.app.LoaderManagerImpl$LoaderObserver.onChanged(LoaderManagerImpl.java:250)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.lifecycle.LiveData.considerNotify(LiveData.java:133)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.lifecycle.LiveData.dispatchingValue(LiveData.java:151)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.lifecycle.LiveData.setValue(LiveData.java:309)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.lifecycle.MutableLiveData.setValue(MutableLiveData.java:50)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.loader.app.LoaderManagerImpl$LoaderInfo.setValue(LoaderManagerImpl.java:189)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.loader.app.LoaderManagerImpl$LoaderInfo.onLoadComplete(LoaderManagerImpl.java:174)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.loader.content.Loader.deliverResult(Loader.java:132)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.philosophicalhacker.lib.RxLoaderCallbacks$RxAndroidLoader.safeDeliverResult(RxLoaderCallbacks.java:109)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.philosophicalhacker.lib.RxLoaderCallbacks$RxAndroidLoader.access$000(RxLoaderCallbacks.java:44)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.philosophicalhacker.lib.RxLoaderCallbacks$RxAndroidLoader$1.accept(RxLoaderCallbacks.java:80)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at io.reactivex.internal.observers.ConsumerSingleObserver.onSuccess(ConsumerSingleObserver.java:62)01-09 09:21:00.006 28092 28092 E AndroidRuntime:at io.reactivex.internal.operators.single.SingleObserveOn$ObserveOnSingleObserver.run(SingleObserveOn.java:81)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at io.reactivex.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:124)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at android.os.Handler.handleCallback(Handler.java:873)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at android.os.Handler.dispatchMessage(Handler.java:99)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at android.os.Looper.loop(Looper.java:214)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at android.app.ActivityThread.main(ActivityThread.java:7063)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at java.lang.reflect.Method.invoke(Native Method)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
01-09 09:21:00.006 28092 28092 E AndroidRuntime: Caused by: com.tspoon.traceur.TraceurException: Debug Exception generated at call site
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at dalvik.system.VMStack.getThreadStackTrace(Native Method)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at java.lang.Thread.getStackTrace(Thread.java:1538)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at io.reactivex.Single.map(Single.java:3008)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.fragment.PullRequestConversationFragment.loadCommitStatusesIfOpen(PullRequestConversationFragment.java:490)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:... 42 more
01-09 09:21:00.006 28092 28092 E AndroidRuntime: Caused by: com.tspoon.traceur.TraceurException: Debug Exception generated at call site
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at dalvik.system.VMStack.getThreadStackTrace(Native Method)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at java.lang.Thread.getStackTrace(Thread.java:1538)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at io.reactivex.Single.map(Single.java:3008)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.fragment.PullRequestConversationFragment.loadCommitStatusesIfOpen(PullRequestConversationFragment.java:491)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:... 42 more
01-09 09:21:00.006 28092 28092 E AndroidRuntime: Caused by: com.tspoon.traceur.TraceurException: Debug Exception generated at call site
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at dalvik.system.VMStack.getThreadStackTrace(Native Method)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at java.lang.Thread.getStackTrace(Thread.java:1538)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at io.reactivex.Single.map(Single.java:3008)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.fragment.PullRequestConversationFragment.loadCommitStatusesIfOpen(PullRequestConversationFragment.java:492)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:... 42 more
01-09 09:21:00.006 28092 28092 E AndroidRuntime: Caused by: com.tspoon.traceur.TraceurException: Debug Exception generated at call site
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at dalvik.system.VMStack.getThreadStackTrace(Native Method)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at java.lang.Thread.getStackTrace(Thread.java:1538)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at io.reactivex.Single.zipArray(Single.java:1974)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at io.reactivex.Single.zip(Single.java:1553)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.fragment.PullRequestConversationFragment.loadCommitStatusesIfOpen(PullRequestConversationFragment.java:505)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:... 42 more
01-09 09:21:00.006 28092 28092 E AndroidRuntime: Caused by: com.tspoon.traceur.TraceurException: Debug Exception generated at call site
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at dalvik.system.VMStack.getThreadStackTrace(Native Method)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at java.lang.Thread.getStackTrace(Thread.java:1538)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at io.reactivex.Single.subscribeOn(Single.java:3675)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.utils.RxUtils.doInBackground(RxUtils.java:89)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.fragment.-$$Lambda$M4WxcKwRhxdwJE_KviDWjmlm4R8.apply(Unknown Source:0)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at io.reactivex.Single.compose(Single.java:2064)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.fragment.LoadingFragmentBase.lambda$makeLoaderSingle$0(LoadingFragmentBase.java:84)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.fragment.LoadingFragmentBase.lambda$makeLoaderSingle$0$LoadingFragmentBase(Unknown Source:0)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.fragment.-$$Lambda$LoadingFragmentBase$ezeDyxD-UcvSxmfC2g4G7aT29vY.apply(Unknown Source:6)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at io.reactivex.Single.compose(Single.java:2064)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.fragment.PullRequestConversationFragment.loadCommitStatusesIfOpen(PullRequestConversationFragment.java:517)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:... 42 more
01-09 09:21:00.006 28092 28092 E AndroidRuntime: Caused by: com.tspoon.traceur.TraceurException: Debug Exception generated at call site
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at dalvik.system.VMStack.getThreadStackTrace(Native Method)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at java.lang.Thread.getStackTrace(Thread.java:1538)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at io.reactivex.Single.observeOn(Single.java:3120)01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.gh4a.utils.RxUtils.doInBackground(RxUtils.java:90)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:... 49 more
01-09 09:21:00.006 28092 28092 E AndroidRuntime: Caused by: com.tspoon.traceur.TraceurException: Debug Exception generated at call site
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at dalvik.system.VMStack.getThreadStackTrace(Native Method)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at java.lang.Thread.getStackTrace(Thread.java:1538)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at io.reactivex.Single.wrap(Single.java:1477)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:... 48 more
01-09 09:21:00.006 28092 28092 E AndroidRuntime: Caused by: com.tspoon.traceur.TraceurException: Debug Exception generated at call site
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at dalvik.system.VMStack.getThreadStackTrace(Native Method)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at java.lang.Thread.getStackTrace(Thread.java:1538)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at io.reactivex.Single.subscribeOn(Single.java:3675)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.philosophicalhacker.lib.ReactiveType$Single.subscribeOn(ReactiveType.java:27)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at com.philosophicalhacker.lib.RxLoaderCallbacks$RxAndroidLoader.onForceLoad(RxLoaderCallbacks.java:76)
01-09 09:21:00.006 28092 28092 E AndroidRuntime:at androidx.loader.content.Loader.forceLoad(Loader.java:347)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at com.philosophicalhacker.lib.RxLoaderCallbacks$RxAndroidLoader.onStartLoading(RxLoaderCallbacks.java:65)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at androidx.loader.content.Loader.startLoading(Loader.java:285)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at androidx.loader.app.LoaderManagerImpl$LoaderInfo.onActive(LoaderManagerImpl.java:77)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at androidx.lifecycle.LiveData.changeActiveCounter(LiveData.java:390)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at androidx.lifecycle.LiveData$ObserverWrapper.activeStateChanged(LiveData.java:466)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at androidx.lifecycle.LiveData$LifecycleBoundObserver.onStateChanged(LiveData.java:425)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at androidx.lifecycle.LifecycleRegistry$ObserverWithState.dispatchEvent(LifecycleRegistry.java:354)01-09 09:21:00.007 28092 28092 E AndroidRuntime:at androidx.lifecycle.LifecycleRegistry.forwardPass(LifecycleRegistry.java:265)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at androidx.lifecycle.LifecycleRegistry.sync(LifecycleRegistry.java:307)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at androidx.lifecycle.LifecycleRegistry.moveToState(LifecycleRegistry.java:148)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at androidx.lifecycle.LifecycleRegistry.handleLifecycleEvent(LifecycleRegistry.java:134)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at androidx.fragment.app.Fragment.performStart(Fragment.java:3026)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at androidx.fragment.app.FragmentStateManager.start(FragmentStateManager.java:589)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:300)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:... 38 more
01-09 09:21:00.007 28092 28092 E AndroidRuntime: Caused by: com.tspoon.traceur.TraceurException: Debug Exception generated at call site
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at dalvik.system.VMStack.getThreadStackTrace(Native Method)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at java.lang.Thread.getStackTrace(Thread.java:1538)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at io.reactivex.Single.observeOn(Single.java:3120)01-09 09:21:00.007 28092 28092 E AndroidRuntime:at com.philosophicalhacker.lib.ReactiveType$Single.observeOn(ReactiveType.java:36)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at com.philosophicalhacker.lib.RxLoaderCallbacks$RxAndroidLoader.onForceLoad(RxLoaderCallbacks.java:77)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:... 53 more
01-09 09:21:00.007 28092 28092 E AndroidRuntime: Caused by: com.tspoon.traceur.TraceurException: Debug Exception generated at call site
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at dalvik.system.VMStack.getThreadStackTrace(Native Method)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at java.lang.Thread.getStackTrace(Thread.java:1538)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at io.reactivex.Single.wrap(Single.java:1477)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at io.reactivex.Single.compose(Single.java:2064)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at com.gh4a.fragment.LoadingFragmentBase.lambda$makeLoaderSingle$0(LoadingFragmentBase.java:85)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at com.gh4a.fragment.LoadingFragmentBase.lambda$makeLoaderSingle$0$LoadingFragmentBase(Unknown Source:0)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at com.gh4a.fragment.-$$Lambda$LoadingFragmentBase$ezeDyxD-UcvSxmfC2g4G7aT29vY.apply(Unknown Source:6)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at io.reactivex.Single.compose(Single.java:2064)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at com.gh4a.fragment.PullRequestConversationFragment.loadCommitStatusesIfOpen(PullRequestConversationFragment.java:517)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at com.gh4a.fragment.PullRequestConversationFragment.onActivityCreated(PullRequestConversationFragment.java:127)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at androidx.fragment.app.Fragment.performActivityCreated(Fragment.java:2996)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at androidx.fragment.app.FragmentStateManager.activityCreated(FragmentStateManager.java:580)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:285)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:... 38 more
01-09 09:21:00.007 28092 28092 E AndroidRuntime: Caused by: com.tspoon.traceur.TraceurException: Debug Exception generated at call site
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at dalvik.system.VMStack.getThreadStackTrace(Native Method)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at java.lang.Thread.getStackTrace(Thread.java:1538)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:at io.reactivex.Single.wrap(Single.java:1477)
01-09 09:21:00.007 28092 28092 E AndroidRuntime:... 44 more
```

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -83,7 +83,7 @@ dependencies {
         exclude group: 'xpp3', module: 'xpp3'
     }
     implementation 'com.github.maniac103:rxloader:master-SNAPSHOT'
-    implementation 'com.github.maniac103:githubsdk:0.7.0.4'
+    implementation 'com.github.maniac103:githubsdk:0.7.0.5'
     implementation 'com.larswerkman:HoloColorPicker:1.5@aar'
     implementation 'com.caverock:androidsvg-aar:1.4'
     implementation 'pl.droidsonroids.gif:android-gif-drawable:1.2.21'

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -73,17 +73,17 @@ dependencies {
     implementation 'androidx.multidex:multidex:2.0.1'
     implementation 'com.google.android.material:material:1.4.0'
     implementation 'io.reactivex.rxjava2:rxandroid:2.1.1'
-    implementation 'com.squareup.retrofit2:adapter-rxjava2:2.5.0'
-    implementation 'com.squareup.retrofit2:converter-moshi:2.5.0'
+    implementation 'com.squareup.retrofit2:adapter-rxjava2:2.6.4'
+    implementation 'com.squareup.retrofit2:converter-moshi:2.6.4'
     implementation 'com.squareup.okhttp3:okhttp:3.12.13'
     implementation 'com.squareup.okhttp3:logging-interceptor:3.12.13'
-    implementation('com.squareup.retrofit2:converter-simplexml:2.5.0') {
+    implementation('com.squareup.retrofit2:converter-simplexml:2.6.4') {
         exclude group: 'stax', module: 'stax-api'
         exclude group: 'stax', module: 'stax'
         exclude group: 'xpp3', module: 'xpp3'
     }
     implementation 'com.github.maniac103:rxloader:master-SNAPSHOT'
-    implementation 'com.github.maniac103:githubsdk:0.7.0.5'
+    implementation 'com.github.maniac103:githubsdk:0.7.0.8'
     implementation 'com.larswerkman:HoloColorPicker:1.5@aar'
     implementation 'com.caverock:androidsvg-aar:1.4'
     implementation 'pl.droidsonroids.gif:android-gif-drawable:1.2.21'

--- a/app/proguard-rules.pro
+++ b/app/proguard-rules.pro
@@ -30,32 +30,6 @@
 -keep class org.ocpsoft.prettytime.i18n.**
 
 #
-# JodaTimeAndroid rules copied from https://github.com/dlew/joda-time-android/blob/main/library/proguard-rules.txt
-#
-
-# All the resources are retrieved via reflection, so we need to make sure we keep them
--keep class net.danlew.android.joda.R$raw { *; }
-
-# These aren't necessary if including joda-convert, but
-# most people aren't, so it's helpful to include it.
--dontwarn org.joda.convert.FromString
--dontwarn org.joda.convert.ToString
-
-# Joda classes use the writeObject special method for Serializable, so
-# if it's stripped, we'll run into NotSerializableExceptions.
-# https://www.guardsquare.com/en/products/proguard/manual/examples#serializable
--keepnames class org.joda.** implements java.io.Serializable
--keepclassmembers class org.joda.** implements java.io.Serializable {
-    static final long serialVersionUID;
-    private static final java.io.ObjectStreamField[] serialPersistentFields;
-    !static !transient <fields>;
-    private void writeObject(java.io.ObjectOutputStream);
-    private void readObject(java.io.ObjectInputStream);
-    java.lang.Object writeReplace();
-    java.lang.Object readResolve();
-}
-
-#
 # Moshi rules copied from https://github.com/square/moshi/blob/moshi-parent-1.8.0/moshi/src/main/resources/META-INF/proguard/moshi.pro
 #
 

--- a/app/src/main/java/com/gh4a/Gh4Application.java
+++ b/app/src/main/java/com/gh4a/Gh4Application.java
@@ -28,8 +28,6 @@ import com.gh4a.worker.NotificationsWorker;
 import com.meisolsson.githubsdk.model.User;
 import com.tspoon.traceur.Traceur;
 
-import net.danlew.android.joda.JodaTimeAndroid;
-
 import org.ocpsoft.prettytime.PrettyTime;
 
 import java.util.HashSet;
@@ -123,7 +121,6 @@ public class Gh4Application extends MultiDexApplication implements
         }
 
         mPt = new PrettyTime();
-        JodaTimeAndroid.init(this);
         ServiceFactory.initClient(this);
 
         updateNotificationWorker(prefs);

--- a/app/src/main/java/com/gh4a/ServiceFactory.java
+++ b/app/src/main/java/com/gh4a/ServiceFactory.java
@@ -33,7 +33,9 @@ import retrofit2.http.POST;
 import retrofit2.http.Path;
 
 public class ServiceFactory {
-    private static final String DEFAULT_HEADER_ACCEPT = "application/vnd.github.v3.full+json";
+    private static final String DEFAULT_HEADER_ACCEPT = "application/vnd.github.v3+json," +
+            "application/vnd.github.v3.raw+json," +
+            "application/vnd.github.v3.html+json";
 
     private final static HttpLoggingInterceptor LOGGING_INTERCEPTOR = new HttpLoggingInterceptor()
             .setLevel(HttpLoggingInterceptor.Level.BASIC);

--- a/app/src/main/java/com/gh4a/activities/IssueEditActivity.java
+++ b/app/src/main/java/com/gh4a/activities/IssueEditActivity.java
@@ -679,10 +679,9 @@ public class IssueEditActivity extends BasePagerActivity implements
     }
 
     private Single<IssueTemplate> parseTemplate(RepositoryContentService service, Content content) {
-        return service.getContents(mRepoOwner, mRepoName, content.path(), null)
+        return service.getContentsRaw(mRepoOwner, mRepoName, content.path(), null)
             .map(ApiHelpers::throwOnFailure)
-            .map(fileContent -> StringUtils.fromBase64(fileContent.content()))
-            .map(contentString -> new IssueTemplate(contentString))
+            .map(IssueTemplate::new)
             .compose(RxUtils::doInBackground);
     }
 

--- a/app/src/main/java/com/gh4a/fragment/CommitFragment.java
+++ b/app/src/main/java/com/gh4a/fragment/CommitFragment.java
@@ -6,6 +6,7 @@ import android.os.Bundle;
 import android.text.SpannableStringBuilder;
 import android.text.TextUtils;
 import android.text.style.ForegroundColorSpan;
+import android.text.style.StrikethroughSpan;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.View.OnClickListener;
@@ -239,7 +240,14 @@ public class CommitFragment extends LoadingFragmentBase implements OnClickListen
 
             ViewGroup fileView = (ViewGroup) inflater.inflate(R.layout.commit_filename, parent, false);
             TextView fileNameView = fileView.findViewById(R.id.filename);
-            fileNameView.setText(file.filename());
+            if (file.previousFilename() != null) {
+                SpannableStringBuilder fileNames = new SpannableStringBuilder();
+                fileNames.append(file.previousFilename()).append('\n').append(file.filename());
+                fileNames.setSpan(new StrikethroughSpan(), 0, file.previousFilename().length(), 0);
+                fileNameView.setText(fileNames);
+            } else {
+                fileNameView.setText(file.filename());
+            }
 
             TextView statsView = fileView.findViewById(R.id.stats);
             if (file.patch() != null) {

--- a/app/src/main/java/com/gh4a/fragment/CommitFragment.java
+++ b/app/src/main/java/com/gh4a/fragment/CommitFragment.java
@@ -188,80 +188,54 @@ public class CommitFragment extends LoadingFragmentBase implements OnClickListen
         LinearLayout llAdded = mContentView.findViewById(R.id.ll_added);
         LinearLayout llRenamed = mContentView.findViewById(R.id.ll_renamed);
         LinearLayout llDeleted = mContentView.findViewById(R.id.ll_deleted);
-        final LayoutInflater inflater = getLayoutInflater();
-        int added = 0, changed = 0, renamed = 0, deleted = 0;
-        int additions = 0, deletions = 0;
-        int count = files != null ? files.size() : 0;
-        int highlightColor = UiUtils.resolveColor(getActivity(), android.R.attr.textColorPrimary);
-        ForegroundColorSpan addSpan = new ForegroundColorSpan(
-                UiUtils.resolveColor(getActivity(), R.attr.colorCommitAddition));
-        ForegroundColorSpan deleteSpan = new ForegroundColorSpan(
-                UiUtils.resolveColor(getActivity(), R.attr.colorCommitDeletion));
-
         llChanged.removeAllViews();
         llAdded.removeAllViews();
         llRenamed.removeAllViews();
         llDeleted.removeAllViews();
 
-        for (int i = 0; i < count; i++) {
+        int addedFiles = 0, changedFiles = 0, renamedFiles = 0, deletedFiles = 0;
+        int totalAdditions = 0, totalDeletions = 0;
+        int filesCount = files != null ? files.size() : 0;
+        int highlightColor = UiUtils.resolveColor(getActivity(), android.R.attr.textColorPrimary);
+        ForegroundColorSpan additionsSpan = new ForegroundColorSpan(
+                UiUtils.resolveColor(getActivity(), R.attr.colorCommitAddition));
+        ForegroundColorSpan deletionsSpan = new ForegroundColorSpan(
+                UiUtils.resolveColor(getActivity(), R.attr.colorCommitDeletion));
+
+        for (int i = 0; i < filesCount; i++) {
             GitHubFile file = files.get(i);
             final LinearLayout parent;
 
             switch (file.status()) {
                 case "added":
                     parent = llAdded;
-                    added++;
+                    addedFiles++;
                     break;
                 case "modified":
                     parent = llChanged;
-                    changed++;
+                    changedFiles++;
                     break;
                 case "renamed":
                     parent = llRenamed;
-                    renamed++;
+                    renamedFiles++;
                     break;
                 case "removed":
                     parent = llDeleted;
-                    deleted++;
+                    deletedFiles++;
                     break;
                 default:
                     continue;
             }
 
-            additions += file.additions();
-            deletions += file.deletions();
+            totalAdditions += file.additions();
+            totalDeletions += file.deletions();
 
-            int commentCount = 0;
-            for (PositionalCommentBase comment : comments) {
-                if (TextUtils.equals(file.filename(), comment.path())) {
-                    commentCount++;
-                }
-            }
-
-            ViewGroup fileView = (ViewGroup) inflater.inflate(R.layout.commit_filename, parent, false);
+            View fileView = getLayoutInflater().inflate(R.layout.commit_filename, parent, false);
             TextView fileNameView = fileView.findViewById(R.id.filename);
-            if (file.previousFilename() != null) {
-                SpannableStringBuilder fileNames = new SpannableStringBuilder();
-                fileNames.append(file.previousFilename()).append('\n').append(file.filename());
-                fileNames.setSpan(new StrikethroughSpan(), 0, file.previousFilename().length(), 0);
-                fileNameView.setText(fileNames);
-            } else {
-                fileNameView.setText(file.filename());
-            }
 
-            TextView statsView = fileView.findViewById(R.id.stats);
-            if (file.patch() != null) {
-                SpannableStringBuilder stats = new SpannableStringBuilder();
-                stats.append("+").append(String.valueOf(file.additions()));
-                int addLength = stats.length();
-                stats.setSpan(addSpan, 0, addLength, 0);
-                stats.append("\u00a0\u00a0\u00a0-").append(String.valueOf(file.deletions()));
-                stats.setSpan(deleteSpan, addLength, stats.length(), 0);
-                statsView.setText(stats);
-                statsView.setVisibility(View.VISIBLE);
-            } else {
-                statsView.setVisibility(View.GONE);
-            }
+            fillFileName(fileNameView, file);
+            fillFileStats(fileView, file, additionsSpan, deletionsSpan);
+            fillFileCommentsCount(fileView, file, comments);
 
             if (file.patch() != null ||
                     (parent != llDeleted && FileUtils.isImage(file.filename()))) {
@@ -269,23 +243,62 @@ public class CommitFragment extends LoadingFragmentBase implements OnClickListen
                 fileView.setOnClickListener(this);
                 fileView.setTag(file);
             }
-            if (commentCount > 0) {
-                TextView commentView = fileView.findViewById(R.id.comments);
-                commentView.setText(String.valueOf(commentCount));
-                commentView.setVisibility(View.VISIBLE);
-            }
 
             parent.addView(fileView);
         }
 
-        adjustVisibility(R.id.card_added, added);
-        adjustVisibility(R.id.card_changed, changed);
-        adjustVisibility(R.id.card_renamed, renamed);
-        adjustVisibility(R.id.card_deleted, deleted);
+        adjustVisibility(R.id.card_added, addedFiles);
+        adjustVisibility(R.id.card_changed, changedFiles);
+        adjustVisibility(R.id.card_renamed, renamedFiles);
+        adjustVisibility(R.id.card_deleted, deletedFiles);
 
         TextView tvSummary = mContentView.findViewById(R.id.tv_desc);
-        tvSummary.setText(getString(R.string.commit_summary, added + changed + renamed + deleted,
-                additions, deletions));
+        tvSummary.setText(getString(R.string.commit_summary,
+                addedFiles + changedFiles + renamedFiles + deletedFiles,
+                totalAdditions, totalDeletions));
+    }
+
+    private void fillFileName(TextView fileNameView, GitHubFile file) {
+        if (file.previousFilename() != null) {
+            SpannableStringBuilder fileNames = new SpannableStringBuilder();
+            fileNames.append(file.previousFilename()).append('\n').append(file.filename());
+            fileNames.setSpan(new StrikethroughSpan(), 0, file.previousFilename().length(), 0);
+            fileNameView.setText(fileNames);
+        } else {
+            fileNameView.setText(file.filename());
+        }
+    }
+
+    private void fillFileStats(View fileView, GitHubFile file, ForegroundColorSpan additionsSpan,
+            ForegroundColorSpan deletionsSpan) {
+        TextView statsView = fileView.findViewById(R.id.stats);
+        if (file.patch() != null) {
+            SpannableStringBuilder stats = new SpannableStringBuilder();
+            stats.append("+").append(String.valueOf(file.additions()));
+            int addLength = stats.length();
+            stats.setSpan(additionsSpan, 0, addLength, 0);
+            stats.append("\u00a0\u00a0\u00a0-").append(String.valueOf(file.deletions()));
+            stats.setSpan(deletionsSpan, addLength, stats.length(), 0);
+            statsView.setText(stats);
+            statsView.setVisibility(View.VISIBLE);
+        } else {
+            statsView.setVisibility(View.GONE);
+        }
+    }
+
+    private void fillFileCommentsCount(View fileView, GitHubFile file,
+            List<? extends PositionalCommentBase> comments) {
+        int commentCount = 0;
+        for (PositionalCommentBase comment : comments) {
+            if (TextUtils.equals(file.filename(), comment.path())) {
+                commentCount++;
+            }
+        }
+        if (commentCount > 0) {
+            TextView commentView = fileView.findViewById(R.id.comments);
+            commentView.setText(String.valueOf(commentCount));
+            commentView.setVisibility(View.VISIBLE);
+        }
     }
 
     private void adjustVisibility(int containerResId, int count) {

--- a/app/src/main/java/com/gh4a/fragment/ContentListContainerFragment.java
+++ b/app/src/main/java/com/gh4a/fragment/ContentListContainerFragment.java
@@ -315,11 +315,10 @@ public class ContentListContainerFragment extends Fragment implements
         String repoOwner = mRepository.owner().login();
         String repoName = mRepository.name();
 
-        service.getContents(repoOwner, repoName, ".gitmodules", mSelectedRef)
+        service.getContentsRaw(repoOwner, repoName, ".gitmodules", mSelectedRef)
                 .map(ApiHelpers::throwOnFailure)
                 .map(Optional::of)
-                .compose(RxUtils.mapFailureToValue(HttpURLConnection.HTTP_NOT_FOUND, Optional.<Content>absent()))
-                .map(contentOpt -> contentOpt.map(content -> StringUtils.fromBase64(content.content())))
+                .compose(RxUtils.mapFailureToValue(HttpURLConnection.HTTP_NOT_FOUND, Optional.<String>absent()))
                 .map(this::parseModuleMap)
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())

--- a/app/src/main/java/com/gh4a/model/StatusWrapper.java
+++ b/app/src/main/java/com/gh4a/model/StatusWrapper.java
@@ -45,6 +45,7 @@ public class StatusWrapper {
         mTargetUrl = checkRun.detailsUrl();
 
         switch (checkRun.state()) {
+            case Waiting:
             case Requested:
             case Queued:
                 mState = State.Unknown;

